python/beeai_framework/adapters/a2a/agents/__init__.py

```diff
@@ -3,11 +3,11 @@
 
 from beeai_framework.adapters.a2a.agents.agent import A2AAgent
 from beeai_framework.adapters.a2a.agents.events import A2AAgentErrorEvent, A2AAgentUpdateEvent
-from beeai_framework.adapters.a2a.agents.types import A2AAgentRunOutput
+from beeai_framework.adapters.a2a.agents.types import A2AAgentOutput
 
 __all__ = [
     "A2AAgent",
     "A2AAgentErrorEvent",
-    "A2AAgentRunOutput",
+    "A2AAgentOutput",
```

Comment:
This is a breaking change. Perhaps I would re-export `A2AAgentRunOutput` as an alias to `A2AAgentOutput` to prevent the current applications from breaking.